src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanics.cpp

```diff
@@ -322,16 +322,18 @@ void SinglePhasePoromechanics< FLOW_SOLVER, MECHANICS_SOLVER >::updateBulkDensit
                                                subRegion );
 }
 
-template class SinglePhasePoromechanics< SinglePhaseBase >;
+template class SinglePhasePoromechanics<>;
 template class SinglePhasePoromechanics< SinglePhaseBase, SolidMechanicsLagrangeContact >;
 template class SinglePhasePoromechanics< SinglePhaseBase, SolidMechanicsEmbeddedFractures >;
```

Comment:
is this necessary? Isn't it already being instantiated in `SinglePhasePoromechanicsConformingFractures.cpp` ?

Reply:
it also seems you are not registering it here.

Reply:
I remember that somehow it was important but I can double-check
Would it miss versions of SinglePhasePoromechanics functions when only present in SinglePhasePoromechanicsConformingFractures?

Reply:
ah yes, sounds possible